examples/rust/publish_subscribe_cross_language/publisher.rs

```diff
@@ -18,6 +18,7 @@ use iceoryx2::prelude::*;
 const CYCLE_TIME: Duration = Duration::from_secs(1);
 
 fn main() -> Result<(), Box<dyn core::error::Error>> {
+    set_log_level_from_env_or(LogLevel::Info);
```

Comment:
I think we can remove this line.

Reply:
see https://github.com/eclipse-iceoryx/iceoryx2/pull/684#discussion_r2042637134